serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/controller/ServiceRecordController.java

```diff
@@ -42,18 +45,22 @@
     private final ServiceRecordFactory serviceRecordFactory;
     private EntityService entityService;
     private PayloadSigner payloadSigner;
+    private SvarUtService svarUtService;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "svarUtService" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1068)